config/routes.rb

```diff
@@ -1,8 +1,10 @@
 Rails.application.routes.draw do
+  get "/ingredients", to:'ingredients#new'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
   get "static_pages/home"
   root "static_pages#home"
+  resources :ingredients
```

Comment:
```suggestion
  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html

  # Defines the root path route ("/")
  # root "articles#index"
  get "static_pages/home"
  root "static_pages#home"
  resources :ingredients, only: %i[index show]
```
I would make the list of ingredients an `index` action rather than a `new` action, since the `new` action implies that you are creating new data.